### ISSUE TITLE
Devops Pipeline, choice type parameter, click on the run button, popbox selection interface has been input box

### ISSUE THREAD

**Issue by xiaohezi205** [kubesphere/ks-devops#537]
Operating System Information：Centos7
Kubernetes version：v1.21.5
KubeSphere version：v3.2.1

question：Devops Pipeline, choice type parameter, click on the run button, popbox selection interface has been input box.The following figure：
![image](https://user-images.githubusercontent.com/84753977/159819895-ca64b5bf-10ac-4a7d-9a36-0490a3cf0d48.png)
![image](https://user-images.githubusercontent.com/84753977/159819942-7d442c29-5080-434d-bbba-2f4ba662962b.png)
![image](https://user-images.githubusercontent.com/84753977/159820074-0fdba8e1-0668-49f1-9e56-622a53bcf395.png)

**Comment by LinuxSuRen** [comment]
I cannot find the corresponding PRs against this issue. But I'm pretty sure this issue was fixed.
/close

**Comment by xiaohezi205** [comment]
我很确认3.3.0，这个bug依然存在，步骤如下：采用的是1master 2node 重新安装的kubesphere3.3.0, 安装完成后，开启了devops，流水线项目，用的是代码根目录下的Jenkinsfile，第一次创建完时，是没有选项的，运行一次加载后，再次运行就是这样明明是choice，但是实际是个string，唉 两个版本了 还是这样

**Comment by LinuxSuRen** [comment]
I'm still not able to reproduce it. See the following screenshot.

![image](https://user-images.githubusercontent.com/1450685/177257396-8acbbe16-64d0-452b-b099-7033738b73bc.png)

See my sample file:

```groovy
pipeline {
  agent any
  stages {
    stage('stage-84kwt') {
      steps {
        echo 'ddd'
      }
    }

  }
  parameters {
    choice(name: 'Environments', choices: ['dev', 'test', 'pre', 'pre2', 'prod'], description: 'desc')
    choice(name: 'ServicesDeploy', choices: ['case-gateway', 'case-auth', 'case-admin', 'case-user', 'case-report', 'case-search', 'case-applet', 'all'], description: 'service')
  }
}
```

**Comment by LinuxSuRen** [comment]
Finally, I found out it has problems when this is a multi-branch pipeline.

**Comment by xiaohezi205** [comment]
奇怪，这次我完全用了你的pipeline脚本，还是没有出现下拉，这是为什么呢？而且我看社区中也有人提了同样的问题。如下图：
![1657000557081](https://user-images.githubusercontent.com/84753977/177259143-7d397c56-59b7-4ec4-a3eb-de06893d01d7.png)
![1657000582732](https://user-images.githubusercontent.com/84753977/177259201-ecce2ec2-458b-41c4-a301-98db1ba43928.png)
![1657000623608](https://user-images.githubusercontent.com/84753977/177259295-aa304908-d897-4c0c-8fb1-00868be024bd.png)

**Comment by wenxin-01** [comment]
Issue fixed in `v3.3.1-rc.1`
